src/compiler/proposals-and-stabilization.md

```diff
@@ -0,0 +1,336 @@
+  - If a team member finds the new attribute reasonable then they can merge the change adding the
+    attribute
+- Adding a attribute with intent to later stabilize
+  - Follow the language team's process and have the implementation PR reviewed by a member of the
+    compiler team
+- Removing internal-use only attributes
+  - **Propose using:** MCP
+  - **Approve using:** Seconding
+  - Describe the rationale for removing the unstable implementation. Once discussion has concluded,
+    a team member may second the proposal
+- Removing attribute which were intended for eventual stabilization
+  - Follow the language team's process and have the removal PR reviewed by a member of the compiler
+    team
+- Stabilizing an attribute
+  - Follow the language team's process and have the stabiization PR reviewed by a member of the
+    compiler team
+- Reverting stabilization of an attribute
+  - Follow the language team's process and have the revert PR reviewed by a member of the
+    compiler team
+
+### Features
+
+- Adding experimental implementations of not-yet-proposed language features
+  - **Propose using:** MCP
+  - **Approve using:** Seconding
+  - With the approval of the language team (that they think the feature is worth experimentation),
+    then submit an RFC and if, after discussion has concluded, a compiler team member agrees that
+    the implementation is feasible and will not put undue burden on the maintainers of the compiler,
+    then they can second the MCP and implementation can proceed.
+  - This isn't necessary if the owner of the implementation is a member of the compiler team
+- Stabilizing a language feature
+  - Follow the language team's process and have the stabiization PR reviewed by a member of the
+    compiler team
+- Reverting stabilization of a language feature
+  - Follow the language team's process and have the revert PR reviewed by a member of the
+    compiler team
+
+### Targets
+
+- Proposing a new target
+  - **Propose using:** PR
+  - **Approve using:** r+ (compiler leads)
+  - Open a PR with the new target (w/ relevant documentation updates) and document adherence to the
+    [target tier policy][tier_policy] in the description. New targets must start as tier three
+  - New targets should be assigned to the compiler team co-leads to check for any licensing
+    concerns
+- Promoting a target
+  - **Propose using:** PR
+  - **Approve using:** r+ (compiler leads)
+  - Open a PR with the new target and document adherence to the [target tier policy][tier_policy]
+    in the description
+  - New targets should be assigned to the compiler team co-leads to ensure that any demands on
+    the project infrastructure are considered and checked with relevant teams
+- Demoting/removing a target
+  - **Propose using:** MCP
+  - **Approve using:** FCP
+  - Write an MCP describing why the target should be demoted/removed and once discussion has
+    concluded, an FCP can be started to approve the demotion/removal.
```

Comment:
Question: is this accurate?

- For instance, demoting Tier 1 -> Tier 2 presently requires an RFC (https://github.com/rust-lang/rfcs/pull/3771), but what about Tier 2 -> Tier 3?
- Also, did you mean Seconding or full FCP (same for Tier 1/2/3 targets respectively?)

Reply:
Fixed!